File: core/src/com/symbol/ecs/component/map/HealthPackComponent.java
package com.symbol.ecs.component.map;

import com.badlogic.ashley.core.Component;
import com.badlogic.gdx.utils.Pool;

public class HealthPackComponent implements Component, Pool.Poolable {

    public int regen = 0;

    @Override
    public void reset() {
        regen = 0;
    }

}

File: core/src/com/symbol/ecs/component/map/HealthPackComponent.kt
package com.symbol.ecs.component.map

import com.badlogic.ashley.core.Component
import com.badlogic.gdx.utils.Pool

class HealthPackComponent : Component, Pool.Poolable {

    var regen = 0

    override fun reset() {
        regen = 0
    }

}